docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1700 @@
+{
```

Comment:
<p>I think everything looks good up until here! Would you like to see if there is anything that you can refactor in <code>discords</code> before I review it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
I reviewed and pushed some commits. I will let you know when I am done so you can review it.